### DISCUSSION TITLE
TSML update not appearing on website in browser of person making the change.

### DISCUSSION THREAD

**Discussion by mikepoconnor4** [code4recovery/12-step-meeting-list#779] [Q&A] [answered]
We’re on wordpress and use the Twelve Step Meeting List plugin to manage our meetings. In the last week we started noticing that when we update a meeting, the person making the change doesn’t see the update on the website but others on other devices will see the update. We cleared the cookies on the browser making the change and then we see the update. This seems to be recent behavior (like the last week or 2) by the plugin - we used to make the change and the person making the change would immediately see the change on the website.

Is anyone else experiencing this? Any insights?

(I originally opened a question in TIAA - dup is here -  https://discourse.tiaa-forum.org/t/twelve-step-meeting-list-plugin/7452)

**Comment by brianw-area24** [comment, marked as answer]
I'm in agreement with the answers on TIAA, and don't know that I can add a lot, other than a little more about the changes that were made.

In version 3.14, we changed the feed that's used by TSML UI to display the meetings. With this change, the feed comes from a file stored on the web server, thus bypassing the overhead of WordPress each time the feed is used. Version 3.14.1 fixed a bug by making a unique filename for the feed. This was needed because without this fix, on multisite installations of WordPress, each site was using the same file and therefore would display the same meetings. With 3.14.1, each site on multisite installations use a different file, therefore only display meetings from that site.

A side effect of this is that the browser cache may need to be cleared when meeting changes are made.

**Reply by joshreisner** [reply]
one thing we might try is appending the file modified time to the url in the query string, eg `/tsml-cache-g468fs.html?123232342146`

this should bust the cache when there's an update while still keeping the benefits of a static file

**Reply by mikepoconnor4** [reply]
I was wondering if the improvement of bypassing the overhead of WordPress is significant - considering that we now have confused office managers (some non-technical) navigating around the caching issues.

**Reply by brianw-area24** [reply]
I think we'll try @joshreisner's suggestion before rolling back to the previous way of doing the feeds. I just created issue #783, so this should be included in the next point release

**Reply by joshreisner** [reply]
thanks brian for adding "cache busting" to address this issue in a recent release. marking as "answered"
